Makefile~

```diff
@@ -0,0 +1,149 @@
+BOOTSTRAP_ARCH = 32
```

Comment:
This temporary file shouldn't be here, should it?

Reply:
Nope, garbage file. I was so happy it was working I just stopped instantly after I did :D